### ISSUE TITLE
maybe you can help me

### ISSUE THREAD

**Issue by pakrett** [EvilJagaGenius/jagoombacolor#8]
Hello ! I just saw that you have made a fork of goomba color so maybe you can help me.
I'm trying to build a modified goomba that use "GBC+SGB" by default instead of "Prefer GBC over SGB" into "Other Settings".
I think there's just a tiny little thing to tweak somewhere, can you help me with that?

**Comment by EvilJagaGenius** [comment]
Line 423 in ui.c should give you the options:
`char *const gbtxt[]={"GB","Prefer SGB over GBC","Prefer GBC over SGB","GBC+SGB"};`
Then line 2834 in gbz80.s should let you change the default.  Change
`	.byte 2 @request_gb_type_`
to
`	.byte 3 @request_gb_type_`
and it should default to GBC+SGB.  Hope this helps!

**Comment by pakrett** [comment]
Oh great, thank you for all ! Do I need something special to build goomba ?

**Comment by EvilJagaGenius** [comment]
No, I don't believe you need anything special for Goomba... you need to rename two files halfway through as specified in the readme, but that's it.  I'm pretty sure my install was standard GBA tools using Msys2.  Can you compile other GBA projects?

**Comment by pakrett** [comment]
Yes I can build the complete firered upgrade but it uses python. I will try tomorrow on another set up using msys2, thank you for helping me ;)

**Comment by pakrett** [comment]
Ok I tried again with msys2 and it works perfectly, thank you again for your help, I'm so happy ! My goal was to launch pokémon crystal clear with SGB borders by default and it works like a charm :)
Sorry to have posted my request here I know that it's not made for that.
